### PR TITLE
Added Problems CRUD and Kendo column

### PR DESCRIPTION
**Pull request by jicata**
Implement functionality for adding problems to Online Contest - closes https://github.com/SoftUni-Internal/suls-issues/issues/3710

### CODE REVIEW

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemsController.cs [hunk 1]
```diff
@@ -934,10 +927,16 @@ private void RetestSubmission(int submissionId)
 
         private DetailedProblemViewModel PrepareProblemViewModelForEdit(int id)
         {
-            return this.Data.Problems.All()
+            var problem = this.Data.Problems.All()
                 .Where(x => x.Id == id)
```
Comment (gogo4ds) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemsController.cs]: use the `ProblemsDataService`

File: Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemsController.cs [hunk 2]
```diff
@@ -934,10 +927,16 @@ private void RetestSubmission(int submissionId)
...
                 .Select(DetailedProblemViewModel.FromProblem)
                 .FirstOrDefault();
+
+            this.AddCheckersAndProblemGroupsToProblemViewModel(
+                problem,
+                this.Data.Problems.GetById(id).Contest.NumberOfProblemGroups);
```
Comment (gogo4ds) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/ProblemsController.cs]: use the `ProblemsDataService`

File: Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Create.cshtml [hunk 3]
```diff
@@ -148,17 +148,39 @@
                 </div>
                 <div class="col-xs-5">
                     @(Html.Kendo()
-                        .DropDownListFor(m => m.Checker)
-                        .Events(e => e.Open("Ojs.KendoControls.DropDownList.alignDropDownToInput"))
-                        .BindTo(Model.AvailableCheckers)
-                        .HtmlAttributes(new { @class = "pull-left full-kendo-editor" }))
+                          .DropDownListFor(m => m.Checker)
```
Comment (Innos) [inline Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/Create.cshtml]: Fix indentation.

File: Open Judge System/Web/OJS.Web/Areas/Administration/ViewModels/Problem/DetailedProblemViewModel.cs [hunk 4]
```diff
@@ -139,6 +141,13 @@ public DetailedProblemViewModel()
         [DefaultValue(0)]
         public int OrderBy { get; set; }
 
+        [ExcludeFromExcel]
+        public IList<SelectListItem> AvailableProblemGroups { get; set; }
```
Comment (Innos) [inline Open Judge System/Web/OJS.Web/Areas/Administration/ViewModels/Problem/DetailedProblemViewModel.cs]: Use the ViewBag instead.

**Pull request merged**